.changeset/many-ghosts-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-app-shortcuts': minor
+---
+
+feat(app-shortcuts): add support for configuration options
```

Comment:
```suggestion
feat: add configuration option
```

FYI: Here we only ever use `android`, `ios` or `web` as the scope. If several platforms are affected, we omit the scope.